Derpy/Program.cs

```diff
@@ -68,6 +70,11 @@ private ServiceProvider LoadDependencies()
                 .AddSingleton<Roles.Service>()
                 .AddSingleton<Tips.Service>()
                 .AddSingleton<Prompt.Service>()
+                .AddSingleton<Palette.Service>()
+                .AddSingleton<HttpMessageHandler>(new HttpClientHandler
+                {
+                    SslProtocols = SslProtocols.Tls12
```

Comment:
Why do we have to change the default here?

Reply:
Ah, forgot this is running on a Linux box, presumably with TLS 1.3. Windows currently only supports TLS 1.2, which is what I'm running, but I'll definitely remove this.